llvm/include/llvm/IR/Type.h

```diff
@@ -334,11 +354,15 @@ class Type {
   /// type.
   unsigned getScalarSizeInBits() const LLVM_READONLY;
 
-  /// Return the width of the mantissa of this type. This is only valid on
-  /// floating-point types. If the FP type does not have a stable mantissa (e.g.
-  /// ppc long double), this method returns -1.
+  /// Return the width of the mantissa of this type, in bits. This is only valid
+  /// on floating-point types. If the FP type does not have a stable mantissa
+  /// (e.g. ppc long double), this method returns -1.
   int getFPMantissaWidth() const;
 
+  /// Return the width of the mantissa of a decimal floating-point type, in
+  /// digits. See table in section X.2.1 of WG14 N2601.
```

Comment:
```suggestion
  /// Return the precision in digits of a decimal floating-point type
  /// per the "Decimal interchange format parameters" table of
  /// C23 annex H.2.1, "Interchange floating types".
```
Let's reference the (not officially complete but close enough) C23 standard here as Joshua suggested.